src/main/resources/META-INF/rewrite/jakarta-ee-10.yml

```diff
@@ -394,12 +394,6 @@ recipeList:
       groupId: jakarta.annotation
       artifactId: jakarta.annotation-api
       newVersion: 2.1.x
-  - org.openrewrite.java.ChangeType:
```

Comment:
ℹ️ The `UpdateJakartaAnnotations2` recipe should not contain these ChangeType changes, as the `JavaxAnnotationMigrationToJakartaAnnotation` recipe, located in jakarta-ee-9.xml, does already do these migrations:

https://github.com/openrewrite/rewrite-migrate-java/blob/main/src/main/resources/META-INF/rewrite/jakarta-ee-9.yml#L94-L129

Because jakarta-ee-10 calls jakarta-ee-9, the namespace fix will already be applied.

---

The reason these ChangeType recipes were added here, was because there was a bug in the `JavaxAnnotationMigrationToJakartaAnnotation` recipe, what let to an unchanged _javax_ namespace. That bug has been [fixed](https://github.com/openrewrite/rewrite-migrate-java/commit/aae42da80d49cdac948c58505925c0d6987d9cb0), so we don't need there recipe here anymore.